src/css/components/_v-scroll-menu-item.scss

```diff
@@ -1,43 +1,67 @@
 .vscrollmenu-item {
     @include display(flex);
+    @include flex-direction(row);
     @include justify-content(space-between);
     @include align-items(center);
-    padding: 50px 55px;
+    padding: 1% 4%;
+    height: calc((#{$master-height} - 75px) / 5);
 
     text-decoration: none;
     color: color(white);
 
     border-bottom: 2px solid;
 
+    &__primary {
+        display: flex;
+        align-items: center;
+
+        width: 100%;
+        height: 100%;
+    }
+
     &__image {
-        @include size(150px);
         margin: 25px auto 12px;
 
-
+        flex: 1;
+        align-self: flex-start;
```

Comment:
```suggestion
```
Removing this allows the image to be centered vertically

Reply:
align-self is required with space-between to not have additional margin on the left. I have added changes to achieve the same effect in https://github.com/smartdevicelink/generic_hmi/pull/253/commits/f76c98926f6963a656f85be07aa1ee186e4f7b68